src/hidpp20.h

```diff
@@ -324,6 +332,12 @@ struct hidpp20_color {
 } __attribute__((packed));
 _Static_assert(sizeof(struct hidpp20_color) == 3, "Invalid size");
 
+enum hidpp20_led_type {
+	HIDPP20_LED_UNKNOWN = -1,
+	HIDPP20_LED_LOGO = 0,
+	HIDPP20_LED_SIDE
```

Comment:
add a trailing comma please, so when we add other things we don't need to change two lines